### PR TITLE
Move refNameAutocomplete out of LGV and into core ui

### PR DESCRIPTION
**Pull request by teresam856**
In this PR: [Issue 2091](https://github.com/GMOD/jbrowse-components/issues/2091)

* Moves the refNameAutocomplete component out of the LGV into the core/ui 
* Adds generic actions to the BaseViewModel for searching capabilities
*  Wires up LGV header and import form with core/ui/RefNameAutocomplete

### CODE REVIEW

File: packages/core/pluggableElementTypes/models/BaseViewModel.ts [hunk 1]
```diff
@@ -14,6 +15,18 @@ const BaseViewModel = types
     menuItems(): MenuItem[] {
       return []
     },
+    currentLocation(): string | undefined {
```
Comment (garrettjstevens) [inline packages/core/pluggableElementTypes/models/BaseViewModel.ts:18]: I think maybe this should be a getter.

```suggestion
    get currentLocation(): string | undefined {
```

Comment (teresam856) [reply packages/core/pluggableElementTypes/models/BaseViewModel.ts:18]: 👍

**Comment by teresam856** [comment]
small demo wiring it up to synteny yeast in branch:  housekeeping_autocomplete_demo 

https://user-images.githubusercontent.com/45598764/128435048-155f14e8-8cbc-4a6c-bbf9-448f8ace2026.mov

**Comment by cmdcolin** [comment]
can close for now, if we get around to this will probably be a fresh attempt

**Pull request closed without merging**